intelmq/lib/upgrades.py

```diff
@@ -550,7 +587,7 @@ def v230_csv_parser_parameter_fix_1(defaults, runtime, harmonization, dry_run):
     ((2, 2, 2), (v222_feed_changes, )),
     ((2, 2, 3), ()),
     ((2, 3, 0), (v230_csv_parser_parameter_fix_1, )),
-    ((3, 0, 0), ()),
+    ((3, 0, 0), (v300_feed_changes, v300_deprecations,)),
```

Comment:
Can you please move the functions to 2.3.0? They are a bug fix IMHO.

Reply:
done